course_contents/2_intro_to_python/notes/02. Loops.md

```diff
@@ -32,50 +35,61 @@ while counter > 0: # the counter never gets changed, causing the conditional to
   print(f"{counter} seconds left until New Year!")
 ```
 
+**Extra resources**
+[While loops](https://www.teclado.com/30-days-of-python/python-30-day-8-while-loops#object-object-loops)
+
 ### The `for` loop
-- for loop runs a given number of times
-- general syntax for `for` loop is:
+- `for` loop runs a given number of times
+- general syntax for using the `for` loop is:
 
 ```py
 for variable in iterable:
```

Comment:
In  the if statements sections you wrote the syntax using angled brackets, like this one:

```
if <expr>:
    <statements>
```

Should we do that here too?

```py
for <variable> in <collection>:
    <statements>  # these run once per item in the collection
```

Reply:
I agree that we should change it, changing and resolving